specs/commandline/key.md

```diff
@@ -115,10 +115,33 @@ notation key list
 
 Upon successful execution, a list of keys is printed out with information of name, key path, certificate path, key id and plugin name. The default signing key name is preceded by an asterisk. The key id and plugin name are used together to provide the information of the key identifier for the remote key and the plugin associated with it.
 
-### Delete two keys from signing key list
+### Remove a specified key from Notation signing key list
 
 ```shell
-notation key delete <key_name_1> <key_name_2>
+notation key remove <key_name>
 ```
 
-Upon successful execution, the names of deleted signing keys are printed out. Please be noted if default signing key is deleted, Notation will not automatically assign a new default signing key. User needs to update the default signing key explicitly.
+- Upon successful removal of a local testing key created by notation, the output message is printed out as follows:
+
+```text
+Removed <key_name> from Notation signing key list. The source key file still exists locally.
+```
+- Upon successful removal of a key associated with a KMS, the output message is printed out as follows:
+
+```text
+Removed <key_name> from Notation signing key list.
+```
+
+### Remove two keys from Notation signing key list
+
+```shell
+notation key remove <key_name_1> <key_name_2>
+```
+
+Upon successful execution, the output message is printed out as follows. Please be noted if default signing key is removed, Notation will not automatically assign a new default signing key. User needs to update the default signing key explicitly.
+
+```text
+Removed the following keys from Notation signing key list. The source key files still exist locally.
```

Comment:
```suggestion
Removed the following keys from Notation signing key list. The source key files still exist.
```
When deleting a key, telling it's a local key or a key stored in a KMS is as tricky as printing out the local key path. Actually, if we could tell a key is a local key, then we are able to print its path out. Both cases need changes on notation CLI + notation-go library.

Reply:
Got it. Keeping minimum changes on the CLI outputs is acceptable without adding additional logic in notation-go lib this time. We could improve this experience thoroughly after v1.0.0 by proposing a feature to clean up local testing keys.